react/components/DomainMessages/index.tsx

```diff
@@ -95,6 +97,10 @@ export const editorMessagesFromRuntime = async ({
   return messagesToReactIntlFormat(messages)
 }
 
+function isValidLang(key: string): key is keyof typeof messages {
+  return Object.prototype.hasOwnProperty.call(messages, key)
```

Comment:
```
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'Messages<"aa" | "ab" | "ae" | "af" | "ak" | "am" | "an" | "ar" | "as" | "av" | "ay" | "az" | "ba" | "be" | "bg" | "bh" | "bi" | "bm" | "bn" | "bo" | "br" | "bs" | "ca" | "ce" | "ch" | "co" | "cr" | ... 156 more ... | "zu">'.
  No index signature with a parameter of type 'string' was found on type 'Messages<"aa" | "ab" | "ae" | "af" | "ak" | "am" | "an" | "ar" | "as" | "av" | "ay" | "az" | "ba" | "be" | "bg" | "bh" | "bi" | "bm" | "bn" | "bo" | "br" | "bs" | "ca" | "ce" | "ch" | "co" | "cr" | ... 156 more ... | "zu">'.ts(7053)
```